### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by fewdan**
### What problem does this PR solve? 
fix typo

### Check List 

Tests 

 - No code

Code changes

 - Has Go code change

Side effects

 - NO

Related changes

 - NO

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: controllers/timechaos/types.go [hunk 1]
```diff
@@ -119,7 +119,7 @@ func (r *Reconciler) Apply(ctx context.Context, req ctrl.Request, chaos reconcil
 	return nil
 }
 
-// Recover means the reonciler recover the chaos action
+// Recover means the reconciler recover the chaos action
```
Comment (yeya24) [inline controllers/timechaos/types.go]: Umm, should this be `reconciler recovers`?

Comment (fewdan) [reply controllers/timechaos/types.go]: done

**Pull request merged**